cmd/version/version.go

```diff
@@ -37,7 +47,25 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 		Use:   "version",
 		Short: "Shows openebs kubectl plugin's version",
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("Client Version: " + rootCmd.Version)
+			k, _ := client.NewK8sClient("")
+			nsMap, err := k.GetVersionMapOfComponents()
+
+			if err != nil {
+				fmt.Println("Client Version: " + rootCmd.Version)
+				fmt.Println("\nError getting Components Version")
```

Comment:
Print errors to stderr via `fmt.FPrintf(os.Stderr, "")`

Reply:
Done.